src/drm/fairplay.js

```diff
@@ -103,14 +114,14 @@ export default class FairPlay extends BaseDrmProtocol {
     try {
       responseObj = JSON.parse(keyText);
     } catch (error) {
-      FairPlay._licenseRequestFailed();
+      FairPlay._errorCallback(new Error(Error.Severity.CRITICAL, Error.Category.DRM, Error.Code.BAD_FAIRPLAY_RESPONSE), error);
     }
     let isValidResponse = FairPlay._validateResponse(responseObj);
     if (isValidResponse.valid) {
       let key = FairPlay._base64DecodeUint8Array(responseObj.ckc);
       FairPlay._keySession.update(key);
     } else {
-      FairPlay._licenseRequestFailed();
+      FairPlay._errorCallback(new Error(Error.Severity.CRITICAL, Error.Category.DRM, Error.Code.BAD_FAIRPLAY_RESPONSE), {data: isValidResponse.reason});
```

Comment:
Here the custom data param called "data" and above called "info"
Let's unified it.

Reply:
i will remove it from here, will add just:
"data / info" in the error class itself.